### ISSUE TITLE
how did you split the csqa dataset

### ISSUE THREAD

**Issue by wuyaoxuehun** [INK-USC/CALM#1]
As stated in your paper, 
"Results for COMMONGEN are on the test set and others are on the official development set. We tune
the hyperparameters based on the models’ performance on a in-house split dev set.", 
so for csqa you split the official train set into train and dev set, right?  please give me more details about the split(ratio, id)
Thank you in advance! Nice work!
